src/components/Nav.js

```diff
@@ -16,8 +16,12 @@ class Nav extends React.Component {
         <div className="nav-1">
           <ul>
             <li className="bookstore">Bookstore&nbsp;CMS</li>
-            <li className="books"><Link to="/">Books</Link></li>
-            <li className="categories"><Link to="/categories">Categories</Link></li>
+            <li className="books">
+              <Link to="/">Books</Link>
+            </li>
+            <li className="categories">
+              <Link to="/categories">Categories</Link>
+            </li>
           </ul>
           <Routes>
             <Route path="/" element={<Book />} />
```

Comment:
- [ ] Kindly make sure you render a Booklist component as required. Hint :point_down: 

```js
// in the BookList component/ Note that this is just an example
.....
 render() {
    return (
      <div>
        <Book title="Book1" author="Author1">
        <Book title="Book2" author="Author2">
        <NewBook />
      </div>
    )
}
.......
```